project 1/styles.css

```diff
@@ -0,0 +1,70 @@
+body {
+    background-color: #DCE6D5;
+}
+
+
+h1 {
+    color: #223809;
+    font-family: 'Libre Baskerville', serif;
+    font-size: 45px;
+    font-weight: 600;
+    line-height: 20px;
+    letter-spacing: .3px;
+    text-align: center;
+}
+
+h2 {
+    color: #223809;
+    font-family: 'Libre Baskerville', serif;
+    font-size: 25px;
+    font-weight: 400;
+    line-height: 15px;
+    letter-spacing: .3px;
+    text-align: center;
+}
+
+p {
+    color: #424242;
+    font-family: 'Libre Baskerville', serif;
+    line-height: 35px;
+    font-size: 15px;
+    font-weight: 500;
+    letter-spacing: .2px;
+    text-align: left;
+}
+
+blockquote {
+    color: #223809;
+    font-family: 'Libre Baskerville', serif;
+    line-height: 40px;
+    font-size: 30px;
+    letter-spacing: .3px;
+    text-align: center;
+}
+
+* {
```

Comment:
I recommend this rule going after your `body` styles -- in general, you want to order things from the most generic/universal to the most specific.